dashboard/app/controllers/api/v1/regional_partners_controller.rb

```diff
@@ -20,6 +20,22 @@ def capacity
     render json: {capacity: get_partner_cohort_capacity(regional_partner_value, role)}
   end
 
+  # GET /api/v1/regional_partners/show/:id
+  def show
+    partner_id = params[:partner_id]
+    begin
+      partner = RegionalPartner.find(partner_id)
+    rescue ActiveRecord::RecordNotFound
+      partner = nil
+    end
```

Comment:
Lines 26-30 can be condensed into one:
```ruby
partner = RegionalPartner.find_by(id: partner_id)
```

Note [find_by](https://apidock.com/rails/v4.0.2/ActiveRecord/FinderMethods/find_by) will return `nil` if the record is not found, whereas [find](https://apidock.com/rails/v4.0.2/ActiveRecord/FinderMethods/find) will raise the error.